src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -245,7 +245,9 @@ private static void register(int id, Set<String> models, BiConsumer<Position, By
         register(89, fmbXXX, (p, b) -> p.set("fuelLevelPercentage", b.readUnsignedByte()));
         register(90, null, (p, b) -> p.set(Position.KEY_DOOR, b.readUnsignedShort()));
         register(110, fmbXXX, (p, b) -> p.set(Position.KEY_FUEL_CONSUMPTION, b.readUnsignedShort() * 0.1));
+        register(111, fmbXXX, (p, b) -> p.set("AdblueLevel", b.readUnsignedByte()));
```

Comment:
The key should be lower camel case, so something like `adBlueLevel` or `adblueLevel`.

Reply:
Also I noticed that the length is not correct. It's not a byte, according to the documentation. Make sure you include a unit test.